sections/all_map_locations.tex

```diff
@@ -115,15 +120,18 @@ \subsection*{Symbols on the Map}
 
   \phantom{j}\textbf{Pandora's Box}\\
   \framedimage[\linewidth]{\map_locations/pandoras_box.jpg}\\
-}
 
-\vspace*{\fill}
-
-\vspace{7.4em}
+  \bigskip
 
-\note{10}{
-  The effects of Fields that allow you to \textbf{Search} for Spells or Artifacts, or where you must spend resources to use the Field's effect, are \textbf{always optional}.
-  You must always place a Black Cube on such Visitable Fields even if you choose not to use that Field's effect.
+  \begin{figure}[H]
+    \begin{minipage}[t]{0.47\textwidth}
```

Comment:
Please don't use that minipage thing, it's a mess. A regular image with `\small` as caption will do.

Reply:
Ah, I just copypasted from below.

Why is minipage used there?

Reply:
Because I did this, when I was starting with the project, and now I'm too lazy to get rid of it.